app/controllers/users_controller.rb

```diff
@@ -151,9 +169,25 @@ def self.calc_calories(preference_list)
 		#seven days a week and also have a physically demanding job, 
 		#multiply by 1.9.
 		#https://www.livestrong.com/article/526442-the-activity-factor-for-calculating-calories-burned/
-		if preference_list[5] == 'Gain'
+		calories = UsersController.goal(calories, preference_list[5])
+		return calories
+	end
+
+	def self.exercise_level(calories, level)
```

Comment:
nice chaining of methods to keep the calc_calories method within the SRP - i had a feeling that the method was doing too much and violating the invariant that the method should really only be concerned with 1 thing.